test/metabase/api/timeline_event_test.clj

```diff
@@ -66,3 +66,20 @@
           (is (true?
                (->> (mt/user-http-request :rasta :put 200 (str "timeline-event/" (u/the-id event)) {:archived true})
                     :archived))))))))
+
+(deftest delete-timeline-event-test
+  (testing "DELETE /api/timeline-event/:id"
+    (mt/with-temp [Collection    collection {:name "Example Data"}
+                   Timeline      timeline   {:name          "Some Events"
+                                             :collection_id (u/the-id collection)}
+                   TimelineEvent event      {:name        "Example Event"
+                                             :timeline_id (u/the-id timeline)}]
+      (let [event-id (u/the-id event)]
+        (testing "delete the timeline-event by `id`"
+          (is (= "Example Event"
```

Comment:
in general, if we want to test the existence of sth we can just fetch it from db directly, like `(t2/select-one-fn :name :model/TimelineEvent :id event-id)`.

But in this case I think we don't need to do that because we explicitly created it with the with-temp above. so this could be removed

Reply:
Good idea, it was created in `with-temp`, so no need to test it.

Remove in https://github.com/metabase/metabase/pull/34072/commits/851ad805eadc347e91088f7e378c7f634c9d9c4d.